common/app/templates/headerInlineJS/shouldEnhance.scala.js

```diff
@@ -4,8 +4,8 @@
 (function (navigator, window) {
     // Enable manual optin to core functionality/optout of enhancement
     var personPrefersCore = function () {
-        if (window.location.hash === '#core' || window.location.hash === 'gu.prefs.force-core=on') return true;
-        if (window.location.hash === '#nocore' || window.location.hash === 'gu.prefs.force-core=off') return false;
+        if (window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
+        if (window.location.hash === '#nocore' || window.location.hash === '#gu.prefs.force-core=off') return false;
```

Comment:
Or could we move the code that saves the preference to above? This seems like the desired behaviour for all preference values.